experiments/one_degree_simulation/one_degree_simulation.jl

```diff
@@ -74,7 +73,7 @@ forcing = (T=FT, S=FS)
 ##### 
 
 momentum_advection = VectorInvariant()
-tracer_advection   = Centered(order=2)
+tracer_advection   = WENO()
```

Comment:
you sure?

Reply:
Nono I reverted to centered. I was trying to see if centered was the issue